### ISSUE TITLE
Documentation export trimmed down in md

### ISSUE THREAD

**Issue by tsunami28** [Azure/enterprise-azure-policy-as-code#743]
**Describe the bug**
Exported md has trimmed values.

**To Reproduce**
Set the policy that allows 3+ VM SKUs and export documentation.

**Expected behavior**
Value in csv is fine:
"{""compute_allowed_disk_type_skus"":""StandardSSD_LRS"",""StandardSSD_ZRS"",""Premium_LRS"",""Premium_ZRS"",""PremiumV2_LRS""]}"
**Screenshots**
![image](https://github.com/user-attachments/assets/40265a0b-4b9f-4d9b-9ce3-858951e48a4a)

**EPAC Version**
Version of EPAC module you are using.
Latest PS module is imported.

**Comment by apybar** [comment]
@tsunami28 - This is actually controlled by the '**markdownMaxParameterLength**' property in your documentation file. If not set, then default is 42 characters. I am updating the documentation online and examples as well. 

See example of my json here:

```
{
    "documentAssignments": {
        "environmentCategories": [
            {
                "pacEnvironment": "EPAC-PROD",
                "environmentCategory": "EPAC-PROD",
                "scopes": [ 
                    "Management Groups: Contoso-Prod"
                ],
                "representativeAssignments": [
                    {
                        "shortName": "test",
                        "id": "/providers/microsoft.management/managementgroups/4fb849a3-3ff3-4362-af8e-45174cd753dd/providers/microsoft.authorization/policyassignments/test"
                    }
                ]
            }
        ],
        "documentationSpecifications": [
            {
                "fileNameStem": "Azure-Policy-Assignments",
                "environmentCategories": [
                    "EPAC-PROD"
                ],
                "title": "Current list of Azure Policies deployed",
                "markdownAdoWiki": true,
                "markdownMaxParameterLength": 1000, // <------------------------------ THIS IS WHAT YOU NEED TO SET :)
                "markdownAdoWikiConfig": [
                    {
                        "adoOrganization": "EPAC",
                        "adoProject": "EPAC",
                        "adoWiki": "EPAC"
                    }
                ]
            }
        ]
    }
}
```

If you are documenting assignments using [environmentCategories](https://azure.github.io/enterprise-azure-policy-as-code/operational-scripts-documenting-policy/#option-2-element-environmentcategories), this should be working on the current release, just set the 'markdownMaxParameterLength' in the 'documentationSpecifications' section to a large number.

If you are using [documentAllAssignments ](https://azure.github.io/enterprise-azure-policy-as-code/operational-scripts-documenting-policy/#option-1-element-documentallassignments), then there currently is a bug however I just resolved it and should be fixed in the next release - I will close this story after the release is published.

**Confirming it is not outputting the full length of the parameter if markdownMaxParameterLength is not set  - or using documentAllAssignments :** 
![image](https://github.com/user-attachments/assets/65f2947a-912f-4303-bea9-6b566516b565)

**I fixed the bug and should be resolved on next release:**
![image](https://github.com/user-attachments/assets/a952ab0a-5fd1-4276-aadf-dc1c41f8da93)

**Comment by anwather** [comment]
Fixed in 10.6.0
